libs/api/domains/auth-admin/src/lib/scope/dto/publish-scope.input.ts

```diff
@@ -0,0 +1,17 @@
+import { Environment } from '@island.is/shared/types'
+import { Field, InputType } from '@nestjs/graphql'
+
+@InputType('AuthAdminPublishScopeInput')
+export class PublishScopeInput {
+  @Field(() => Environment, { nullable: false })
```

Comment:
```suggestion
  @Field(() => Environment)
```

Nullable false is the default. Same for below.